errors/errors.go

```diff
@@ -34,4 +34,8 @@ var (
 	ErrConfigNotFound          = errors.New("cli: config with the given name does not exist")
 	ErrNoURLProvided           = errors.New("cli: no URL provided in argument or via config. see 'zot config -h'")
 	ErrIllegalConfigKey        = errors.New("cli: given config key is not allowed")
+	ErrScanNotSupported        = errors.New("search: squashfs image scanning not supported")
+	ErrJSONNotFound            = errors.New("repository: index.json not found")
```

Comment:
https://github.com/anuvu/zot/blob/master/pkg/storage/storage.go#L264
we already have an error for this case, reuse it

Reply:
I added this error because ErrJSONNotFound gives more detailed reason "repository: index.json not found" instead of "repository: not found".

Reply:
Without an index.json, not really a valid repo for us. So just reuse that same pattern, and add logs.

Reply:
Ok, I will do